ui/browser/reducers/main-reducers.js

```diff
@@ -135,3 +142,10 @@ function setCurrentTab(state, pageIndex) {
 function setPageOrder(state, pageOrder) {
   return state.set('pageOrder', pageOrder);
 }
+
+function togglePagesSection(state, opened) {
```

Comment:
And in the case where we noop because we're attempting to open an already opened pages tab, then nothing will happen in our views as state didn't really change